LenovoLegionToolkit.WPF/Windows/Settings/NotificationsSettingsWindow.xaml.cs

```diff
@@ -42,6 +44,8 @@ public NotificationsSettingsWindow()
             _powerModeToggle.IsChecked = _settings.Store.Notifications.PowerMode;
             _refreshRateToggle.IsChecked = _settings.Store.Notifications.RefreshRate;
             _acAdapterToggle.IsChecked = _settings.Store.Notifications.ACAdapter;
+            _smartKeySinglePressToggle.IsChecked = _settings.Store.Notifications.SmartKeySinglePress;
```

Comment:
I would cover both single and double press in a single setting. Otherwise looks great.

Reply:
I thought someone might come and say "But I want them separate!".

Reply:
haha, yes, but I will just say no :D